### ISSUE TITLE
[iOS] Deep Link & Push Notifications Infrastructure Setup

### ISSUE THREAD

**Issue by moiz994** [openedx/openedx-app-ios#247]
Implementation of the required infrastructure to easily implement Deep Links and Push notifications with third-party tools.

**Comment by marcotuts** [comment]
We have some duplicate tickets here we should centralize - a deep linking and push notification epic exists for iOS and Android in addition to this fyi @moiz994

**Comment by moiz994** [comment]
Yes, I just saw the duplicates. Is there a way to delete the old issues?

**Delete/Close:** https://github.com/openedx/openedx-app-ios/issues/116, https://github.com/openedx/openedx-app-android/issues/118 
**Keep:** https://github.com/openedx/openedx-app-ios/issues/247, https://github.com/openedx/openedx-app-android/issues/201

**Comment by edschema** [comment]
@moiz994 @touchapp @saeedbashir closing this epic as all sub issues are complete.
